src/app/components/Text/index.jsx

```diff
@@ -13,6 +13,24 @@ const StyledParagraph = styled.p`
     line-height: 1.375rem;
   }
 `;
+const StyledItalic = styled.i`
+  // Font styling below is a subset of BBC GEL Typography "Body Copy"
+  font-size: 0.9375em;
+  line-height: 1.25em;
```

Comment:
The line-height values now need to be in `rem` instead of `em`, following this task being completed (https://github.com/bbc/simorgh/issues/407).  If you could change the unit on this line and on lines 22, 28, 31 that would be great. Thanks.

Reply:
Sure. Wasn't aware of this when I started this ticket. Thanks.

Reply:
You shouldn't need to change these styles, see my comment https://github.com/bbc/simorgh/pull/477#discussion_r210207083.